packages/webview_flutter/webview_flutter_wkwebview/pigeons/web_kit.dart

```diff
@@ -849,6 +853,35 @@ abstract class NSUrlFlutterApi {
   void create(int identifier);
 }
 
+/// Host API for `UIScrollViewDelegate`.
+///
+/// This class may handle instantiating and adding native object instances that
+/// are attached to a Dart instance or method calls on the associated native
+/// class or an instance of the class.
+///
+/// See https://developer.apple.com/documentation/uikit/uiscrollviewdelegate?language=objc.
+@HostApi(dartHostTestHandler: 'TestUIScrollViewDelegateHostApi')
+abstract class UIScrollViewDelegateHostApi {
+  @ObjCSelector('createWithIdentifier:')
+  void create(int identifier);
+}
+
+/// Flutter API for `UIScrollViewDelegate`.
+///
+/// See https://developer.apple.com/documentation/uikit/uiscrollviewdelegate?language=objc.
+@FlutterApi()
+abstract class UIScrollViewDelegateFlutterApi {
+  @ObjCSelector(
+    'scrollViewDidScrollWithIdentifier:uiScrollViewIdentifier:x:y:',
```

Comment:
Nit: ObjC naming should be `...:UIScrollViewIdentifier:x:y:`. Capitalizing acronyms trumps first letter being lower case for camel case in Obj-C naming conventions.

Reply:
Lemme fix that.